ihmc-ros2-library/build.gradle.kts

```diff
@@ -15,6 +15,8 @@ mainDependencies {
    api("us.ihmc:ihmc-pub-sub:0.19.0")
    api("us.ihmc:ihmc-realtime:1.6.0")
    api("us.ihmc:ihmc-commons:0.32.0")
+
+   api("us.ihmc:ros2-common-interfaces:source")
```

Comment:
How come this guy is needed?

Reply:
That's for the change to `newMessageTopicDataTypeInstance`. To send the primitives as messages themselves, you need this. I think it was overlooked initially because most people don't do that.

Reply:
I can't imagine the case where you'd somehow want to use the ihmc-ros2-library, but not the common msgs included with ROS and the Euclid type support. Seems silly to protect that use case.